voucher/src/main/res/layout/full_voucher_view.xml

```diff
@@ -91,51 +93,54 @@
         app:layout_constraintTop_toBottomOf="@id/paymentReferenceSeparator"
         tools:text="12345.1234.12345.12345.12345678901 123456789012345" />
 
-    <com.google.android.material.button.MaterialButton
-        android:id="@+id/button_copyCode"
-        style="@style/AdyenCheckout.Voucher.ButtonCopyCode"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        app:layout_constraintEnd_toStartOf="@+id/button_downloadPdf"
-        app:layout_constraintHorizontal_chainStyle="packed"
-        app:layout_constraintStart_toStartOf="parent"
+    <Space
+        android:id="@+id/space_buttons"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/standard_margin"
         app:layout_constraintTop_toBottomOf="@id/textView_reference_code" />
 
-    <com.google.android.material.button.MaterialButton
-        android:id="@+id/button_downloadPdf"
-        style="@style/AdyenCheckout.Voucher.ButtonDownloadPdf"
-        android:layout_width="wrap_content"
+    <com.google.android.flexbox.FlexboxLayout
```

Comment:
What do we need this for exactly?

Reply:
When button labels are big, we need to move the second button below the first button. `FlexboxLayout` solves that issue, but it might not be exactly what we want.
I am thinking of removing it and handling that case with a custom code.

What we want is, when the buttons are on one line, they need to be aligned to the start and the end of the view. But when they are above one another, they need to be centered.

Reply:
This is actually possible with just a `ConstraintLayout` by using the `Flow` tag. For example something like this:
```XML
<androidx.constraintlayout.helper.widget.Flow
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        app:constraint_referenced_ids="button1,button2"
        app:flow_horizontalStyle="spread_inside"
        app:flow_wrapMode="aligned"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintTop_toTopOf="parent" />
```